src/latch_cli/services/register/staging.py

```diff
@@ -211,14 +211,12 @@ def register_staging(
     skip_confirmation: bool = False,
     wf_module: Optional[str] = None,
     progress_plain: bool = False,
+    dockerfile_path: Optional[Path] = None,
 ):
     wf_module = wf_module if wf_module is not None else "wf"
     module_path = pkg_root / Path(wf_module.replace(".", "/"))
 
-    dockerfile_path = pkg_root / "Dockerfile"
```

Comment:
seems like this would break backwards compatibility? maybe we need to keep this?